### PR TITLE
Fixes #24386 - Update debug to include proxy certs generate.

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@lzap can I get a review please?

File: script/foreman-debug [hunk 1]
```diff
@@ -352,6 +352,7 @@ add_files /var/log/foreman/db_seed*.log*
 add_files /var/log/foreman/production-*.log*
 add_files /var/log/foreman/production.log
 add_files /var/log/foreman/dynflow_executor.output*
+add_files /var/log/foreman-proxy-certs-generate.log*
```
Comment (lzap) [inline script/foreman-debug]: It should be actually this: `/var/log/foreman-proxy-certs-generate.*log`

**Review by lzap — CHANGES_REQUESTED** [review]
There is slightly different naming convention as you'd expect:

```
[root@sat-r220-07 ~]# ll /var/log/foreman-proxy-certs-generate.*log
-rw-r--r--. 1 root root 117278 Apr 30 11:37 /var/log/foreman-proxy-certs-generate.10.log
-rw-r--r--. 1 root root 131204 Apr 30 11:23 /var/log/foreman-proxy-certs-generate.11.log
-rw-r--r--. 1 root root 130025 Apr 30 11:02 /var/log/foreman-proxy-certs-generate.12.log
-rw-r--r--. 1 root root      0 Mar 17 15:26 /var/log/foreman-proxy-certs-generate.13.log
-rw-r--r--. 1 root root  21157 Mar 17 15:18 /var/log/foreman-proxy-certs-generate.14.log
-rw-r--r--. 1 root root  21150 Dec  4  2017 /var/log/foreman-proxy-certs-generate.15.log
-rw-r--r--. 1 root root      0 Dec  4  2017 /var/log/foreman-proxy-certs-generate.16.log
-rw-r--r--. 1 root root      0 Dec  4  2017 /var/log/foreman-proxy-certs-generate.17.log
-rw-r--r--. 1 root root      0 Dec  4  2017 /var/log/foreman-proxy-certs-generate.18.log
-rw-r--r--. 1 root root 129104 May  9 11:06 /var/log/foreman-proxy-certs-generate.1.log
-rw-r--r--. 1 root root 131227 May  9 10:43 /var/log/foreman-proxy-certs-generate.2.log
-rw-r--r--. 1 root root 129455 Apr 30 16:39 /var/log/foreman-proxy-certs-generate.3.log
-rw-r--r--. 1 root root      0 Apr 30 15:23 /var/log/foreman-proxy-certs-generate.4.log
-rw-r--r--. 1 root root 117279 Apr 30 15:22 /var/log/foreman-proxy-certs-generate.5.log
-rw-r--r--. 1 root root 116673 Apr 30 13:42 /var/log/foreman-proxy-certs-generate.6.log
-rw-r--r--. 1 root root      0 Apr 30 12:47 /var/log/foreman-proxy-certs-generate.7.log
-rw-r--r--. 1 root root  89711 Apr 30 12:33 /var/log/foreman-proxy-certs-generate.8.log
-rw-r--r--. 1 root root 117279 Apr 30 12:33 /var/log/foreman-proxy-certs-generate.9.log
-rw-r--r--. 1 root root 128777 May  9 11:06 /var/log/foreman-proxy-certs-generate.log
```

**Comment by chris1984** [comment]
@lzap Updated, good catch!

**Comment by chris1984** [comment]
[test foreman]

**Comment by chris1984** [comment]
[test katello]

**Review by lzap — APPROVED** [review]
TY

**Pull request merged**